lib/converters/v2.0.0/converter-v2-to-v1.js

```diff
@@ -433,7 +449,9 @@ _.assign(Builders.prototype, {
             retainEmpty = self.options.retainEmptyValues,
             urlObj = _.isString(url) ? util.urlparse(url) : url;
 
-        !skipResponses && units.push('responses');
+        if (!skipResponses) {
+            units = units.concat(['responses', 'responses_order']);
```

Comment:
I feel two `push`s here are faster than `concat` because it's creating a new Array and copying all the items.